src/libraries/Microsoft.Extensions.Logging.Console/src/SimpleConsoleFormatter.cs

```diff
@@ -158,12 +162,8 @@ private static string GetLogLevelString(LogLevel logLevel)
 
         private ConsoleColors GetLogLevelConsoleColors(LogLevel logLevel)
         {
-            bool isAndroidOrAppleMobile = RuntimeInformation.IsOSPlatform(OSPlatform.Create("ANDROID"))
-                                          || RuntimeInformation.IsOSPlatform(OSPlatform.Create("TVOS"))
-                                          || RuntimeInformation.IsOSPlatform(OSPlatform.Create("IOS"))
-                                          || RuntimeInformation.IsOSPlatform(OSPlatform.Create("MACCATALYST"));
             bool disableColors = (FormatterOptions.ColorBehavior == LoggerColorBehavior.Disabled) ||
-                (FormatterOptions.ColorBehavior == LoggerColorBehavior.Default && (!ConsoleUtils.EmitAnsiColorCodes || isAndroidOrAppleMobile));
+                (FormatterOptions.ColorBehavior == LoggerColorBehavior.Default && (!ConsoleUtils.EmitAnsiColorCodes || _isAndroidOrAppleMobile));
```

Comment:
Could you please add a comment telling why we are doing `_isAndroidOrAppleMobile` checking? just for the code readers later understand what is going on. Thanks!

Reply:
Done